kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -65,15 +71,47 @@
 	protected List<DisbursementPayee> getPersonAsPayeesByEmployeeStatusCode(Map<String, String> fieldValues, String employeeStatusCode) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		fieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
-		List<? extends Person> persons = KimApiServiceLocator.getPersonService().findPeople(fieldValues);
+		Map<String, String> personFieldValues = this.getPersonFieldValues(fieldValues);
+		personFieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
+		List<? extends Person> persons = getPersonService().findPeople(personFieldValues);
 
 		for (Person personDetail : persons) {
-			DisbursementPayee payee = getPayeeFromPerson(personDetail, fieldValues);
+			DisbursementPayee payee = getPayeeFromPerson(personDetail, personFieldValues);
 			payeeList.add(payee);
 		}
 
 		return payeeList;
 	}
 
+	private static volatile PersonService personService;

```

Comment:
What's the reasoning here for the volatile keyword? I don't see the need for memory visibility, am I missing something? Also our convention is to keep all instance and class variables declared at the top of the class.